client/src/index.tsx

```diff
@@ -3,8 +3,6 @@ import ReactDOM from "react-dom";
 import App from "./components/App";
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <App />,
```

Comment:
Oh so this gets rid of that warning in the console regarding "strict mode" ?

Reply:
Yup, is that ok?